src/programs/project3d/project3d.cpp

```diff
@@ -129,7 +147,29 @@ bool Project3DApp::DoCalculation()
 	cisTEMParameterLine input_parameters;
 
 	cisTEMParameters input_star_file;
-	input_star_file.ReadFromcisTEMStarFile(input_star_filename);
+	if (project_based_on_star == true) input_star_file.ReadFromcisTEMStarFile(input_star_filename);
+
+	EulerSearch	global_euler_search;
+	ParameterMap parameter_map; // needed for euler search init
+	parameter_map.SetAllTrue();
+
+	if (project_based_on_star == false)
+	{
+
+		global_euler_search.InitGrid(my_symmetry, angular_step, 0.0f, 0.0f, 360, 180, 180, pixel_size / 8, parameter_map, 20);
+		//	wxPrintf("%s",my_symmetry);
+		if (my_symmetry.StartsWith("C1")) // TODO 2x check me - w/o this O symm at least is broken
```

Comment:
Is the TODO still to do? Just flagging this in case you just forgot to remove the TODO comment

Reply:
Got rid of TODO and added some more comments

Reply:
There is a similar line in match_template - please check to see if your modifications should apply there as well.

https://github.com/timothygrant80/cisTEM/blob/0aabb63c9cc6197150891606e85719f4ea9f57b7/src/programs/match_template/match_template.cpp#L690-L697

Reply:
This is the same between the two.  I wrote the match _template one originally and copied it here.

Reply:
@timothygrant80 What happens if the search is CX? Shouldn't the mirror also be skipped?

E.g., Running match_template with C2 prints ~4x fewer search positions than C1, which seems to me like a problem.

Reply:
Yes, I think you are correct.. it should be if (my_symmetry.StartsWith("C"))

Reply:
Updated this